src/sequenceformats/csv2fasta/config.vsh.yaml

```diff
@@ -0,0 +1,92 @@
+name: csv2fasta
+description: Convert two columns from a CSV file to FASTA entries.
+argument_groups:
+  - name: Inputs
+    arguments:
+    - name: --input
+      type: file
+      direction: input
+      example: barcodes.csv
+      description: CSV file to be processed.
+      required: true
+  - name: "CSV Format arguments"
+    arguments:
+      - name: --header
+        type: boolean_true
+        description: |
+          Parse the first line of the CSV file as a header.
+      - name: --delimiter
+        type: string
+        description: |
+          Column delimiter
+        default: ","
+      - name: --quote_character
+        type: string
+        description: |
+          Character used to denote the start and end of a quoted item.
+        default: '"'
+  - name: "CSV column arguments"
+    description: |
+      Parameters for the selection of columns from the CSV file.
+      Only required when your CSV file contains more than 2 columns.
+    arguments:
+      - name: --sequence_column
```

Comment:
Question: is 'sequence' clear enough? What about 'barcode'?

Reply:
'Sequence' is more general, a DNA sequence can be a barcode, a primer, a genomics sequence,... anything realy. It can even be an RNA sequence.